Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutAccountNonce.swift

```diff
@@ -3,7 +3,9 @@ import BraintreePayPal
 #endif
 
 /// Contains information about a PayPal payment method.
-@objc public class BTPayPalNativeCheckoutAccountNonce: NSObject {
+@objcMembers public class BTPayPalNativeCheckoutAccountNonce: NSObject {
```

Comment:
Noticed that none of these properties were actually exposed to Obj-C. Added `objcMembers` so that all public properties in the none are now exposed as expected.